packages/binding-http/test/http-server-test.ts

```diff
@@ -549,6 +549,100 @@ class HttpServerTest {
         return httpServer.stop();
     }
 
+    @test async "should report allproperties"() {
```

Comment:
```suggestion
    @test async "should report allproperties excluding no json properties"() {
```
or create two tests with one with only json properties and one like the current version.

Reply:
I had to re-read it multiple times to get it right.. but I think the addition should read
"excluding non-JSON properties"